tox.ini

```diff
@@ -16,24 +17,21 @@ passenv =
     LANG
     HOME
 commands =
-    pip install -e .
-    pytest --cov=cookiecutter --cov-report=term --cov-fail-under=100 --cov-branch {posargs:tests}
+    poetry run pytest --cov=cookiecutter --cov-report=term --cov-fail-under=100 --cov-branch {posargs:tests}
     cov-report: coverage html
     cov-report: coverage xml
-deps = -rtest_requirements.txt
 skip_install = true
+whitelist_externals = poetry
 
 [testenv:lint]
 commands =
-    python -m pre_commit run {posargs:--all}
+    poetry run python -m pre_commit run {posargs:--all}
 deps = pre-commit>=1.20.0
 usedevelop = false
 
 [testenv:docs]
 commands =
-    pip install -e .
-    pip install -r test_requirements.txt
-    pip install -r docs/requirements.txt
+    poetry run pip install -r docs/requirements.txt
```

Comment:
Just noting that Poetry will add the concept of [Dependency groups in 1.2](https://python-poetry.org/blog/announcing-poetry-1.2.0a2/#dependency-groups) which might be a good place to track documentation dependencies.

I've been watching this for a while but sadly it's still in alpha...

Reply:
this is nice to have..  thank you for sharing...